pwndbg/commands/search.py

```diff
@@ -299,6 +305,11 @@ def search(
     ):
         if save:
             saved.add(address)
+        if type == "asm" and asmbp:
+            ins = pwndbg.disasm.one(address)
```

Comment:
Why do we need to disasm it?

Reply:
oh sorry I overlooked the redundancy while making changes